docs/source/channel_update_tutorial.rst

```diff
@@ -280,7 +249,7 @@ Now we will make use of the ``configtxlator`` tool to decode this channel
 configuration block into JSON format (which can be read and modified by humans).
 We also must strip away all of the headers, metadata, creator signatures, and
 so on that are irrelevant to the change we want to make. We accomplish this by
-means of the ``jq`` tool:
+means of the ``jq`` tool (you will need to install the `jq tool <https://stedolan.github.io/jq/>`_ on your local machine):
```

Comment:
Might be best to move this dependency to a before you begin section at the beginning, similar to this section in the write your first app tutorial? https://hyperledger-fabric.readthedocs.io/en/latest/write_first_app.html#before-you-begin

Reply:
This is consistent with the create channel tutorial. I'd suggest we open a new JIRA to handle this separately for both tutorials.

Reply:
Created [FAB-18394](https://jira.hyperledger.org/browse/FAB-18394)

Reply:
ok